CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.12)
+project (lvgl)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2 -s USE_SDL=2")
+
+include_directories(${PROJECT_SOURCE_DIR})
+
+file(GLOB INCLUDES1 "./*.h")
+file(GLOB_RECURSE INCLUDES2 "lv_drivers/*.h" "lv_examples/*.c" "lvgl/*.h")
```

Comment:
Should be ""lv_examples/*.c"", isn't it?

Reply:
Fixed